app/models/message_draft.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Message drafts are essentially messages with an implied folder (drafts)
+class MessageDraft < Message
+end
```

Comment:
Oddly enough I recall create draft and update draft failing without subject, body, recipient and body - but I will check again